src/platform_impl/android/mod.rs

```diff
@@ -344,22 +344,58 @@ impl EventLoop {
                         };
 
                     for pointer in pointers {
-                        let location =
+                        let position =
                             PhysicalPosition { x: pointer.x() as _, y: pointer.y() as _ };
                         trace!(
-                            "Input event {device_id:?}, {phase:?}, loc={location:?}, \
+                            "Input event {device_id:?}, {phase:?}, loc={position:?}, \
                              pointer={pointer:?}"
                         );
+                        let finger_id = event::FingerId(FingerId(pointer.pointer_id()));
+                        let force = Some(Force::Normalized(pointer.pressure() as f64));
 
-                        let event = event::WindowEvent::Touch(event::Touch {
-                            device_id,
-                            phase,
```

Comment:
Now that we no longer output `phase`, does it make any sense to map to `TouchPhase`? This would only be used in the case of gestures which are not even implemented yet, perhaps we can remove this side-stepping?

Reply:
Done!